### ISSUE TITLE
YAML experiment definitions seem not to support adaptation of echam namelist parameter putrerun

### ISSUE THREAD

**Issue by christian-stepanek** [esm-tools/esm_tools#113]
For long simulations it is not useful to store monthly restarts, this unnecessarily bloats disk usage.

Setting the namelist parameter putrerun: [1, 'years', 'first', 0], in order to store restarts only once a year, worked in the past when editing esm-runscripts directly. Yet, if trying to set the parameter change in the YAML-Syntax as follows
```
echam:
    namelist_changes:
        namelist.echam:
            putrerun: [1, 'years', 'first', 0]
```
there is apparently a problem with namelist.py that drops the following traceback:
```
Traceback (most recent call last):
  File "/pf/a/a270061/.local/bin/esm_runscripts", line 8, in <module>
    sys.exit(main())
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_runscripts/cli.py", line 168, in main
    Setup()
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_runscripts/esm_sim_objects.py", line 54, in __call__
    self.compute(*args, **kwargs)
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_runscripts/esm_sim_objects.py", line 80, in compute
    self.config = Compute.evaluate(self.config)
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_runscripts/jobclass.py", line 24, in evaluate
    config = esm_plugin_manager.work_through_recipe(self.framework_recipe, self.framework_plugins, config)
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_plugin_manager/esm_plugin_manager.py", line 96, in work_through_recipe
    config = getattr(submodule, workitem)(config)
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_runscripts/compute.py", line 105, in modify_namelists
    config[model] = namelist.nmls_remove(config[model])
  File "/pf/a/a270061/.local/lib/python3.6/site-packages/esm_runscripts/namelists.py", line 41, in nmls_remove
    value = change_entries[key]
```
As YAML seems to become the suggested standard for experiment configuration and the ability to change restart writing frequency is of high importance, I propose consider this issue high priority.

**Comment by pgierz** [comment]
OK, I found the problem: Try setting `restart_rate: "year"` under the echam section of your yaml file. Changing AWICM settings apparently don't carry over. That needs to be fixed. Also, the other method should also be supported...

However, if the addition to the yaml file works, I suggest we use that for now.

**Comment by christian-stepanek** [comment]
This indeed works, at least the esm-tools do not drop any error anymore:
```
echam:
    restart_rate: "year"
    namelist_changes:
        namelist.echam:
            putrerun: [1, 'years', 'first', 0]
```
This should go into an updated example script file, I make a new issue for that.

**Comment by pgierz** [comment]
The second part can be removed:
```
    namelist_changes:
        namelist.echam:
            putrerun: [1, 'years', 'first', 0]
```

so, it should **only** be:
```
echam:
    restart_rate: "year"
```
